lib/Db/MemberMapper.php

```diff
@@ -183,4 +185,43 @@ public function updateMemberNameByUserId(string $userId, string $displayName): v
 			->where($qb->expr()->eq('user_id', $qb->createNamedParameter($userId)));
 		$qb->executeStatement();
 	}
+	
+	/**
+	 * Set user_id to null when it points to a user that does not exist
+	 *
+	 * @param string|null $projectId If set, only try to fix members of this project
+	 * @return void
+	 * @throws Exception
+	 */
+	public function cleanupUserMembers(?string $projectId = null): void {
```

Comment:
Why filter by project id at all? It's easier to just fix every project in one go, especially if a user is mapped to multiple members.

Reply:
With the goal of doing it often, it made sense to do it even if the info of one project was requested.
But if we go with a repair step, no need to filter. :+1: